libs/ledger-live-common/src/families/stellar/api/horizon.ts

```diff
@@ -107,24 +121,25 @@ export const fetchAccount = async (
   spendableBalance: BigNumber;
   assets: BalanceAsset[];
 }> => {
-  let account: typeof AccountRecord = {};
-  let balance: Record<string, any> = {};
+  let account: ServerApi.AccountRecord = {} as ServerApi.AccountRecord;
```

Comment:
I don't understand why you force cast `{}` as `account` is typed.

Reply:
It's because we initialized the value just below, here we just making sure that the variable is not undefined